lte/gateway/c/oai/oai_mme/oai_mme.c

```diff
@@ -73,7 +75,7 @@ static int main_init(void) {
   init_task_context(
       TASK_MAIN,
       (task_id_t[]){TASK_MME_APP, TASK_SERVICE303, TASK_SERVICE303_SERVER,
-                    TASK_S6A, TASK_S1AP, TASK_SCTP, TASK_SPGW_APP,
+                    TASK_S6A, TASK_S1AP, TASK_NGAP, TASK_SCTP, TASK_SPGW_APP,
```

Comment:
remove TASK_NGAP from this list and activate the task optionally based on a converged core config parameter in mme.yml and mme.conf.template files (check stateless or HA services as examples)